Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/clerk.yml

```
@@ -11,4 +11,6 @@
     - id: BoxEncryptionKeyJustice
     - id: ClerkIDCard
     - id: RubberStampNotary
+    - id: LPPRubberStampConfidentional #LPP
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)